index.js

```diff
@@ -165,7 +165,7 @@ const fastifyReplyFrom = fp(function from (fastify, opts, next) {
         if (customRetry && customRetry.handler) {
           const customRetries = customRetry.retries || 1
           if (++retries < customRetries) {
-            return customRetry.handler(req, res, defaultDelay)
+            return customRetry.handler(req, res, err, defaultDelay)
```

Comment:
Since it will be braking change, I wonder if an object is better so users will have a smooter transition and we could add more info in future if needed (such as the num of retry) without shipping it as breaking

Reply:
I don't think it is a breaking change because the original PR hasn't been released yet -- it has just been sitting on main!

Reply:
Oh nice!

Do you think the object interface could be better?

Reply:
You mean like a destructuring approach where you can only pull out the properties you want?

Reply:
Yes, this one:

```suggestion
            return customRetry.handler({ req, res, err, defaultDelay })
```

Reply:
I figure that is a little bit less performant as it adds another allocation and restructuring, and IMO 4 args is a lot but its not so many that we need to do this. If we suspected that we were going to add more properties over time and wanted a way to do that without making a breaking change, I'd say that's a great idea, but I don't think there's much more information that is going to exist about an error that we would want to pass in in the future. If you feel strongly I am ok with it.

Reply:
The most common use I see:
after 4 retries I want to log the message, otherwise ignore it.

For this reason I think this object will change soonish

Reply:
Can you explain more? Are you saying we should add the attempt count?

Reply:
If we break, what about putting the error to be in the first place?

Reply:
It is not a breaking change, this feature is not yet released

Reply:
This has been changed to return an object that we can extend over time!